packages/api/src/command/medical/document/start-bulk-get-doc-url.ts

```diff
@@ -37,45 +33,41 @@ export const startBulkGetDocumentUrls = async (
     log(
       `Patient ${patientId}, Request ${bulkGetDocUrlProgress?.requestId}, bulkGetDocUrlProgress is already 'processing', skipping...`
     );
-    return createBulkGetDocumentUrlQueryResponse("processing", patient);
+    return createBulkGetDocumentUrlQueryResponse(BulkGetDocUrlStatus.processing, patient);
   }
 
   const requestId = getOrGenerateRequestId(bulkGetDocUrlProgress);
 
   const updatedPatient = await storeBulkGetDocumentUrlQueryInit({
     id: patient.id,
     cxId: patient.cxId,
-    bulkGetDocumentsUrlProgress: { status: "processing" },
+    status: BulkGetDocUrlStatus.processing,
     requestId,
   });
 
-  const payload: DocumentBulkSignerLambdaRequest = {
+  const payload: DocumentBulkSignerRequest = {
     patientId: patientId,
     cxId: cxId,
     requestId: requestId,
   };
 
+  const documentBulkSigner = makeDocumentBulkSigner();
```

Comment:
using factory. the lambda signer has a .catch on the promise like we discussed and then rethrows the error, so the try catch here will catch it

Reply:
got it :)